src/goto-harness/recursive_initialization.cpp

```diff
@@ -280,6 +321,19 @@ bool recursive_initializationt::should_be_treated_as_array(
          initialization_config.pointers_to_treat_as_arrays.end();
 }
 
+optionalt<std::size_t>
```

Comment:
Small preference to have a (strong?) typedef for this `size_t` so we know that it refers to an “equal cluster”

Reply:
Done.